Doc/reference/datamodel.rst

```diff
@@ -2419,15 +2442,16 @@ Emulating callable objects
 
    Called when the instance is "called" as a function; if this method is defined,
    ``x(arg1, arg2, ...)`` roughly translates to ``type(x).__call__(x, arg1, ...)``.
-
+   The :class:`object` class itself does not provide this method.
 
 .. _sequence-types:
 
 Emulating container types
 -------------------------
 
-The following methods can be defined to implement container objects.  Containers
-usually are :term:`sequences <sequence>` (such as :class:`lists <list>` or
+The following methods can be defined to implement container objects. None of them
+are provided by the :class:`object` class itself. Containers usually are
+:term:`sequences <sequence>` (such as :class:`lists <list>` or
 :class:`tuples <tuple>`) or :term:`mappings <mapping>` (like
 :class:`dictionaries <dict>`),
```

Comment:
```suggestion
:class:`dicts <dict>`),
```

This is emitted as a literal, so it should be the literal name (or very close to it)

Reply:
Friendly disagreement here!  IMO it’s fine to have the class formatting even if we’re using the full term _dictionaries_.
It’s a normal word to read, we get a link to more info, the formatting is not a problem.

Reply:
Friendly disagreement with the friendly disagreement here :) Since this is displayed as a literal, that by definition implies that it is the _literal_ code name of the class referenced, which is of course not the case. However, its not the end of the world; we can agree to disagree here and leave it to @terryjreedy to decide either way?

Reply:
I think your approach is to ask whether the markup is 100% philosophically correct.
Mine is to ask whether there is a problem (in function, like links, or confusion for readers), which I really don’t think there is with literal style on a non-code term here.

Reply:
Hmm, well it indeed does seem unlikely that (most) readers won't be able to make the connection, true, even though it is technically incorrect. However, even so, it still may appear to be a formatting mistake, it dilutes the well-established meaning of monospaced text in docs for readers, and results in a moment of cognitive dissonance for readers familiar with it that takes. These costs may be relatively trivial for a single instance, but readers constantly having to process such inconsistencies can add up to a non-trivial mental toll reducing overall docs quality, particularly for reference documentation such as this. Indeed, [Diataxis stresses this point repeatedly](https://diataxis.fr/reference/):

> Your users need reference material because they need truth and certainty - firm platforms on which to stand while they work.

> There should be no doubt or ambiguity in reference; it should be wholly authoritative.

> Reference material benefits from consistency. Be consistent, in structure, language, terminology, tone.

> These descriptions must be accurate [...] any discrepancy between the machinery and your description of it will inevitably lead a user astray.

Also, conceptually, `dict` refers to the concrete Python _class_, while "dictionary" refers to the abstract Python data structure/protocol—which are separate references in the docs. If we want to refer to the latter instead here, we can link [the appropriate glossary term](https://docs.python.org/3.11/glossary.html#term-dictionary) (and presumably should do the same for [list](https://docs.python.org/3.11/glossary.html#term-list):

```suggestion
:term:`dictionaries <dictionary>`),
```